test/generators/go/GoGenerator.spec.ts

```diff
@@ -155,4 +155,31 @@ type CustomEnum string`;
     expect(enumModel.result).toEqual(expected);
     expect(enumModel.dependencies).toEqual([]);
   });
+  test('should render models and their dependencies', async () => {
```

Comment:
I don't see dependencies being set in this test.

Reply:
Changed 🙂